db/seeds.rb

```diff
@@ -5,3 +5,8 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+Lifehack.create([{
+title: 'How to tie a tie',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.